todo_app/app.py

```diff
@@ -3,40 +3,221 @@
+		"GET",
+		url,
+		headers=headers,
+		params=query,
+		
+	)
 	
-	return redirect(url_for('index'))
 	
-@app.route('/save', methods =['POST','GET'])
-def update():
-   #if item is saved then item.status= completed
-    if request.method == 'POST':
-	    get_id = request.form.get('itemId')
+    
+	return response
+
+def getDoing():
+	url = f"https://api.trello.com/1/lists/{doingId}/cards"
+
+	headers = {
+		"Accept": "application/json"
+	}
+
+	query = {
+	    'id' : doingId,
+		'key': os.environ['KEY'],
+		'token' : os.environ['TOKEN']
+	}
+
+	responseDoing = requests.request(
+		"GET",
+		url,
+		headers=headers,
+		params=query
+	)
+	
+    
+	return responseDoing
+	
+def getDone():
+	url = f"https://api.trello.com/1/lists/{doneId}/cards"
+
+	headers = {
+		"Accept": "application/json"
+	}
+
+	query = {
+	    'id' : doneId,
+		'key': os.environ['KEY'],
+		'token' : os.environ['TOKEN']
+	}
+
+	responseDone = requests.request(
+		"GET",
+		url,
+		headers=headers,
+		params=query
+	)
+	
+    
+	return responseDone
+	
+def newToDoCard(name):
+	url = "https://api.trello.com/1/cards"
+
+	query = {
+		'key': os.environ['KEY'],
+		'token' : os.environ['TOKEN'],
+		'idList': '6005828032dafa5707bf5dc5',
+		'name': name
+	}
+
+	response = requests.request(
+		"POST",
+		url,
+		params=query
+	)
+def updateCardToDone(i):
+	url = f"https://api.trello.com/1/cards/{i}"
+
+	headers = {
+		"Accept": "application/json"
+	}
+
+	query = {
+	    'id': i,
+		'key': os.environ['KEY'],
+		'token' : os.environ['TOKEN'],
+		'idBoard': '6005828032dafa5707bf5dc3',
+		'idList': '6005828032dafa5707bf5dc7'
+		
+	}
+
+	response = requests.request(
+		"PUT",
+		url,
+		headers=headers,
+		params=query
+	)
+
+def returnCardToDo(i):
+		url = f"https://api.trello.com/1/cards/{i}"
+
+		headers = {
+			"Accept": "application/json"
+		}
+
+		query = {
+			'id': i,
+			'key': os.environ['KEY'],
+			'token' : os.environ['TOKEN'],
+			'idBoard': '6005828032dafa5707bf5dc3',
+			'idList': '6005828032dafa5707bf5dc5'
+		
+		}
+
+		response = requests.request(
+			"PUT",
+			url,
+			headers=headers,
+			params=query
+	)
+
+
+
+@app.route('/')
+
+def index():
+ response = getToDo()
+ responseDoing = getDoing()
+ responseDone = getDone()
+ jsonResponse = response.json()
+ jsonResponseDoing = responseDoing.json()
+ jsonResponseDone = responseDone.json()
+ number = len(jsonResponse)
+ numberTwo = len(jsonResponseDoing)
+ numberThree = len(jsonResponseDone)
+ 
+ my_objects = []
+ 
+ for listNumber in range(number):
 	
-	    items = get_item(get_id)
-	    updated = save_item(items)
-		#x = sorted(updated)
-   
-    return redirect(url_for('index'))
+	 my_objects.append(ToDo(jsonResponse[listNumber]['id'],jsonResponse[listNumber]['name']))
+	 
+	 
+ return render_template("index.html", jsonResponse = jsonResponse, jsonResponseDoing = jsonResponseDoing,jsonResponseDone = jsonResponseDone, number = number, numberTwo = numberTwo, numberThree = numberThree, my_objects = my_objects)
```

Comment:
`jsonResponse` and `number` are no longer used by index.html because you are just using `my_objects`